Dockerfile

```diff
@@ -84,8 +84,8 @@ RUN groupadd -r sdcli -g 1000 \
 
 FROM user_deps AS docker_cli_deps
 # https://docs.docker.com/engine/install/debian/
-ENV DOCKER_PACKAGE_VERSION=5:20.10.6~3-0~debian-bullseye
-ENV COMPOSE_PLUGIN_PACKAGE_VERSION=2.11.2~debian-bullseye
+ENV DOCKER_PACKAGE_VERSION=5:27.3.1-1~debian.11~bullseye
+ENV COMPOSE_PLUGIN_PACKAGE_VERSION=2.29.7-1~debian.11~bullseye
 ENV COMPOSE_PACKAGE_VERSION=1.25.0-1
```

Comment:
I am considering dropping `sdcli` as a concept., but would like to bring it to a known good state for "legacy" projects where it can stick around for a while.